pkg/shares/parse_compact_shares.go

```diff
@@ -3,20 +3,31 @@ package shares
 import (
 	"encoding/binary"
 	"errors"
+	"fmt"
 
 	"github.com/celestiaorg/celestia-app/pkg/appconsts"
 )
 
-// parseCompactShares takes raw shares and extracts out transactions,
+// parseCompactShares takes rawShares and extracts out transactions,
 // intermediate state roots, or evidence. The returned [][]byte do not have
 // namespaces, info bytes, data length delimiter, or unit length
 // delimiters and are ready to be unmarshalled
-func parseCompactShares(shares [][]byte) (data [][]byte, err error) {
```

Comment:
[Non Blocking][Question]
Where is this actually called? I found it being used in `func Merge(eds...)`, but I don't see this latter being used anywhere

Reply:
> Where is this actually called?

1. https://github.com/rootulp/celestia-app/blob/24ddfce6c6b523f4e0dc5de6dd780bc3ad5ed272/pkg/shares/share_merging.go#L80
2. https://github.com/rootulp/celestia-app/blob/24ddfce6c6b523f4e0dc5de6dd780bc3ad5ed272/pkg/shares/share_merging.go#L80

which get called from `func Merge(eds...)` like you said. I also don't see `Merge` used anywhere outside tests

Reply:
As far as I understand, this was moved from core to app as it was mainly used in app. And, this was done prior to non-interactive defaults as intended to be used by that. 
I don't want to mention Evan as he's on vacation and if I mention him, he will respond :D

Reply:
Agreed. It seems non-blocking to find usage of this `Merge(eds...)` function so I think this can merge irrespective of investigating that further

Reply:
Should we create an issue for that so that we remmeber it? or just keep it in mind until Evan is back?
I am fine with any. Approooving

Reply:
Created https://github.com/celestiaorg/celestia-app/issues/840